packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.6
+
+* iOS: `webViewWebContentProcessDidTerminate` is recognized in `WebView.onWebResourceError`.
```

Comment:
nit: I found this description somewhat hard to understand, maybe something like: "Invoke the `WebView.onWebResourceError` on iOS when the webview content process crashes".

[assuming `webViewWebContentProcessDidTerminate` is only invoked when the process terminates due to some error, is this the case?]

Reply:
done!

[The documentation didn't specify what reasons the termination could be. I assume it is related to some sort of crashes. In the case I tried to fix, it was related to memory pressure]